packages/x-components/src/components/animations/staggered-fade-and-slide.vue

```diff
@@ -48,22 +48,26 @@
 <style lang="scss" scoped>
   $transition-duration: 0.25s;
 
-  .x-staggered-fade-and-slide::v-deep .x-staggered-fade-and-slide {
-    &--enter-active,
-    &--leave-active {
-      transition: $transition-duration ease-out;
-      transition-property: opacity, transform;
-    }
+  .x-staggered-fade-and-slide {
+    z-index: 0;
 
-    &--move {
-      transition: transform $transition-duration ease-out;
-      z-index: 1;
-    }
+    &::v-deep .x-staggered-fade-and-slide {
+      &--enter-active,
+      &--leave-active {
+        transition: $transition-duration ease-out;
+        transition-property: opacity, transform;
+      }
+
+      &--move {
+        transition: transform $transition-duration ease-out;
```

Comment:
Actually, we can maintain the `z-index: 1;` if the container has `z-index: 0;`. Whatever you prefer

Reply:
I buy the change like this.
Is this to remove the `1001` index that you added in motive?

Reply:
Nope, it is to live together with the `z-index` of the `base-id-modal` component. https://github.com/empathyco/x/blob/main/packages/x-components/src/components/modals/base-modal.vue#L277